src/components/Layout/LandingPage.jsx

```diff
@@ -1,27 +1,42 @@
 import React from 'react';
-import './Layout.css';
-
 
 const LandingPage = () => (
-  <div className="container">
-    <div className="box" id="box1">
-      <h3><strong>DevConnection:</strong> Personal Journey into the World of Professional Development</h3>
-      <h5>Welcome to DevConnection — a platform created specifically for aspiring developers, web designers, and testers, among others. Here, at the intersection of technology and creativity, you embark on your exciting journey into professional development.</h5>
+  <div class="flex flex-row items-center h-screen bg-black pb-28">
+    <div className="basis-1/2">
+      <div className="w-3/4 mx-auto">
+        <h3 className="text-2xl mb-4 text-blue"><span className="font-bold text-blue">DevConnexion:</span> Personal Journey into the World of Professional Development</h3>
+        <p className="font-sans font-light mb-8">Welcome to DevConnection — a platform created specifically for aspiring developers, web designers, and testers, among others. Here, at the intersection of technology and creativity, you embark on your exciting journey into professional development.</p>
+        <input type="text"  placeholder="Search your first project here" className="search-area w-3/4"></input> 
+      </div>
     </div>
-    <div className="box" id="box2">
-      <div className="container2">
-        <h1>Sign In</h1>
-        <div className="login">
-          Login: <br />
-          <input type="text" className="landingPage-input" placeholder="Email or username"></input> <br />
-          Password: <br />
-          <input type="text" className="landingPage-input" placeholder="Password"></input> <br />
-          <button className="signInOnLandingPage">Sign In</button>
-          <h6>Don't have an account? <a href="/register">Sign Up</a></h6>
+    <div className="basis-1/2">
+      <div className="w-3/5 mx-auto px-8 ">
+        <h1 className="text-center text-xl pb-6">Sign In</h1>     
+        <input type="text"  placeholder="Email" className=" input-area  mb-4"></input> 
+        <input type="text"  placeholder="Password" className="input-area mb-4"></input>
```

Comment:
Let's change this to `<input type="password" placeholder="Password" className="input-area mb-4"></input>`
![image](https://github.com/Code-the-Dream-School/ffprac-team1-front/assets/42615340/e55930cc-3170-45ef-8c7f-7ac480f54079)